client/state/sites/domains/test/fixture/index.js

```diff
@@ -47,6 +47,7 @@ export const DOMAIN_PRIMARY = {
 	isPendingIcannVerification: false,
 	isPendingWhoisUpdate: false,
 	isSubdomain: false,
+	isWpcomStagingDomain: false,
```

Comment:
General observation: these tests don't look like they include cases where we _do_ have a `wpcomstaging.com` domain. That should probably be its own Asana card so we can try to prevent bugs from creeping in moving forward.

cc @klimeryk

Reply:
From my experience, these Calypso tests don't actually test anything useful, so 🤷‍♂ Even if, it's not on us to ensure proper wpcomstaging.com subdomain behaviour, IMO.